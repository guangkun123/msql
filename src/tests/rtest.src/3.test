# Do some lookups 

select phone from test where user = 'Bambi'\p\g
select phone from test where user LIKE '%ambi'\p\g
select phone from test where age < 25\p\g
select phone from test where age > 25\p\g
select phone from test where user > 'apple'\p\g
select * from test where phone = '(075) 951412' \p\g

select * from test2 \p\g
select user from test2 where fullname = NULL\p\g
select user from test2 where fullname <> NULL\p\g
