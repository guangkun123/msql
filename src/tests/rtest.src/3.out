
Welcome to the miniSQL monitor.  Type \h for help.


mSQL >     -> 
Query buffer
------------

select phone from test where user = 'Bambi'
[continue]
    -> 
Query OK.

1 rows matched.

 +----------------------+
 | phone                |
 +----------------------+
 | (075) 951450         |
 +----------------------+



mSQL > 
Query buffer
------------
select phone from test where user LIKE '%ambi'
[continue]
    -> 
Query OK.

1 rows matched.

 +----------------------+
 | phone                |
 +----------------------+
 | (075) 951450         |
 +----------------------+



mSQL > 
Query buffer
------------
select phone from test where age < 25
[continue]
    -> 
Query OK.

0 rows matched.

 +----------------------+
 | phone                |
 +----------------------+
 +----------------------+



mSQL > 
Query buffer
------------
select phone from test where age > 25
[continue]
    -> 
Query OK.

9 rows matched.

 +----------------------+
 | phone                |
 +----------------------+
 | (075) 951450         |
 | (075) 951431         |
 | 99                   |
 | 97                   |
 | 95                   |
 | 93                   |
 | 91                   |
 | 89                   |
 | 87                   |
 +----------------------+



mSQL > 
Query buffer
------------
select phone from test where user > 'apple'
[continue]
    -> 
Query OK.

9 rows matched.

 +----------------------+
 | phone                |
 +----------------------+
 | (075) 951412         |
 | (075) 951431         |
 | 99                   |
 | 97                   |
 | 95                   |
 | 93                   |
 | 91                   |
 | 89                   |
 | 87                   |
 +----------------------+



mSQL > 
Query buffer
------------
select * from test where phone = '(075) 951412' 
[continue]
    -> 
Query OK.

1 rows matched.

 +------------+----------+----------------------+
 | user       | age      | phone                |
 +------------+----------+----------------------+
 | paulp      | 25       | (075) 951412         |
 +------------+----------+----------------------+



mSQL >     -> 
Query buffer
------------
select * from test2 
[continue]
    -> 
Query OK.

4 rows matched.

 +------------+--------------------------------+
 | user       | fullname                       |
 +------------+--------------------------------+
 | paulp      | Paul Pyyvaara                  |
 | Bambi      | David J. Hughes                |
 | fred       | Fred O'Niell                   |
 | Nobody     | NULL                           |
 +------------+--------------------------------+



mSQL > 
Query buffer
------------
select user from test2 where fullname = NULL
[continue]
    -> 
Query OK.

1 rows matched.

 +------------+
 | user       |
 +------------+
 | Nobody     |
 +------------+



mSQL > 
Query buffer
------------
select user from test2 where fullname <> NULL
[continue]
    -> 
Query OK.

3 rows matched.

 +------------+
 | user       |
 +------------+
 | paulp      |
 | Bambi      |
 | fred       |
 +------------+



mSQL > 
Bye!

